lib/run.go

```diff
@@ -131,9 +131,16 @@ func (a *ACBuild) Run(cmd []string, insecure bool) (err error) {
 	}
 
 	if man.App != nil {
+		setPath := false
 		for _, evar := range man.App.Environment {
+			if evar.Name == "PATH" {
```

Comment:
How does rkt handle when a manifest sets a path variable then? Does it just ignore it?